server/src/internalClusterTest/java/org/elasticsearch/snapshots/SnapshotStatusApisIT.java

```diff
@@ -428,8 +428,11 @@ public void testGetSnapshotsWithSnapshotInProgress() throws Exception {
                 return false;
             }
             var shards = snapshotsInProgress.snapshot(snapshots.iterator().next()).shards();
-            long initShards = shards.stream().filter(e -> e.getValue().state() == SnapshotsInProgress.ShardState.INIT).count();
-            long successShards = shards.stream().filter(e -> e.getValue().state() == SnapshotsInProgress.ShardState.SUCCESS).count();
+            long initShards = shards.entrySet().stream().filter(e -> e.getValue().state() == SnapshotsInProgress.ShardState.INIT).count();
+            long successShards = shards.entrySet()
```

Comment:
Optional thing, but if you want this can be changed to `.values()` instead of `.entrySet()` since the key is never used